### PR TITLE
[bug] ensures that repository link is available on npm

### PR DESCRIPTION
**Pull request by gabrielcsapo**
Screen Shot 2020-09-22 at 5 57 42 PM

repo link didn't exist on npmjs.com

### CODE REVIEW

**Comment by rwjblue** [comment]
😩 looks like CI has issues

File: lib/resolve-package-path.d.ts [hunk 1]
```diff
@@ -1,12 +1,9 @@
 import Cache = require('./cache');
 import CacheGroup = require('./cache-group');
-declare function _getRealFilePath(realFilePathCache: Cache, filePath: string): string | null;
-declare function _getRealDirectoryPath(realDirectoryPathCache: Cache, directoryPath: string): string | null;
-declare function _findPackagePath(realFilePathCache: Cache, name: string, dir: string): string | null;
```
Comment (rwjblue) [inline lib/resolve-package-path.d.ts]: Hmm, I do find it easier to read these as stand alone `declare function` calls. Why move them down inline?

**Review by rwjblue — CHANGES_REQUESTED** [review]
This package is using `yarn`, can you update the `yarn.lock` (and remove the `package-lock.json`)?

Let's split things up a bit here:

* PR to update `repository` field in package.json
* PR to update `resolve` to latest
* PR to update the rest of the `devDependencies`

**Comment by gabrielcsapo** [comment]
@rwjblue updated to just be the repository link fix in this PR.

**Review by rwjblue — APPROVED** [review]
Awesome, thank you!

**Pull request merged**